src/components/modules/chat/ui/components/ChatInterface.tsx

```
@@ -0,0 +1,119 @@
+
+  // Group messages by date
+  const groupedMessages = messages.reduce((groups, message) => {
+    const date = message.timestamp.toDateString();
+    if (!groups[date]) {
+      groups[date] = [];
+    }
+    groups[date].push(message);
+    return groups;
+  }, {} as Record<string, ChatMessageType[]>);
+
+  const formatDate = (dateString: string) => {
```

ESLint

[error] 'formatDate' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)